includes/defaults.inc.php

```diff
@@ -31,6 +31,8 @@
 
 // MySQL extension to use
 $config['db']['extension']       = 'mysqli';//mysql and mysqli available
+// MySQL Debug level
+$config['mysql_log_level']       = 'ERROR';
```

Comment:
Set this as the default so all installs will log errors. We could set it to NONE but imho this is now handy to find non-working things easily and won't spam the log file anyway.